src/select/__integ__/select.test.ts

```diff
@@ -109,8 +109,6 @@ test(
     const { height: actualDropdownHeight } = await page.getBoundingBox(optionsSelector);
     const availableDropdownHeight = smallestContainerHeight - triggerHeight;
     expect(actualDropdownHeight).toBeLessThan(availableDropdownHeight);
-    const { top: containerScrollTop } = await page.getElementScroll('#smallest_container');
-    expect(containerScrollTop).toBe(0);
```

Comment:
that check was here since the beginning (`CR-16802930`, but it's not even when it was introduced, just refactored to `.ts`). 
It tries to scroll an element with `overflow: hidden` and I don't really understand the intention, so I just deleted